compiler/rustc_ast_passes/src/feature_gate.rs

```diff
@@ -505,6 +505,7 @@ pub fn check_crate(krate: &ast::Crate, sess: &Session, features: &Features) {
     gate_all!(contracts, "contracts are incomplete");
     gate_all!(contracts_internals, "contract internal machinery is for internal use only");
     gate_all!(where_clause_attrs, "attributes in `where` clause are unstable");
+    gate_all!(super_let, "`super let` is experimental");
```

Comment:
I can't tell whether this list wants to be alphabetized or not.  It doesn't seem chronological.

Reply:
Well it's certainly not alphabetical, so it's unclear where else I should put this other than just at the end.

Reply:
Yes, it's OK.